apps/src/turtle/turtle.js

```diff
@@ -275,23 +277,38 @@ Artist.prototype.init = function (config) {
  * orientation.
  */
 Artist.prototype.prepareForRemix = function () {
-  if (REMIX_PROPS.every(group => Object.keys(group.defaultValues).every(prop =>
+  const blocksDom = Blockly.Xml.blockSpaceToDom(Blockly.mainBlockSpace);
+  const blocksDocument = blocksDom.ownerDocument;
+  let next, removedBlock = false;
+  let setArtistBlock = blocksDom.querySelector('block[type="turtle_setArtist"]');
+  while (setArtistBlock) {
+    removedBlock = true;
+    next = setArtistBlock.querySelector('next');
+    let parentNext = setArtistBlock.parentNode;
+    let parentBlock = parentNext.parentNode;
+    parentBlock.removeChild(parentNext);
+    if (next) {
+      parentBlock.appendChild(next);
+    }
+    setArtistBlock = blocksDom.querySelector('block[type="turtle_setArtist"]');
+  }
```

Comment:
When you remix a frozen share, it gets converted to a standard artist project. I assumed we only wanted Elsa and Anna to be available on frozen levels.

If that's not the case, there's probably more work to be done to get Anna and Elsa working on levels that didn't start out as frozen, but I haven't tried it yet.